prow/presubmit.yaml

```diff
@@ -76,8 +76,10 @@ kubernetes/kubernetes:
 
 - name: pull-kubernetes-cross
   context: Jenkins Cross Build
-  rerun_command: "@k8s-bot build this"
-  trigger: "@k8s-bot (cross )?build this"
+  always_run: true
+  skip_report: true
+  rerun_command: "@k8s-bot cross test this"
```

Comment:
The problem is that the mergebot says `@k8s-bot test this`, so all `always_run` jobs need to match that.